### DISCUSSION TITLE
dry-buckets for wells

### DISCUSSION THREAD

**Discussion by Matt-PMCT** [DFHack/dfhack#3219] [Ideas]
I spend an annoying amount of my game time building/destroying wells. I find that the "bucket is full" in the well and the dwarfs won't use the well. The only thing that seems to fix it is to disassemble the well, rebuilding it with a different bucket.

The dry-bucket command states "Buckets in wells or being actively carried or used by a job are not affected." Is there any way to create a dry well bucket command, because that would really help me from having to build/destroy/rebuild wells...

**Comment by lethosor** [comment]
This seems like a solid suggestion to me, so I'm moving it to the issue tracker where we also track suggestions: #3227
